imgix/urlbuilder.py

```diff
@@ -50,14 +51,16 @@ def __init__(
             domain,
             use_https=True,
             sign_key=None,
-            include_library_param=True):
+            include_library_param=True,
+            disable_variable_quality=False):
```

Comment:
To stay consistent with all other implementations so far, we should remove this from the class constructor and add it as an option to `create_srcset`. Something to the effect of:

```python
def create_srcset(
            self, path, params={},
            start=MIN_WIDTH, stop=MAX_WIDTH, tol=TOLERANCE, disable_variable_quality=False):
```

I know that looks clunky, so I'm open to different naming. But you get the idea.

Reply:
I also wouldn't mind seeing a validator for the `disable_variable_quality` flag as per [imgix-rb:L180](https://github.com/imgix/imgix-rb/blob/d5964b86ebb1c4611dab2aeb69703d28e68c264e/lib/imgix/path.rb#L180). But I will defer to your decision.

Reply:
Okay yeah, I could add this validator. Just to check that the boolean flag is a boolean right?

Also, as far as the snippet above, and keeping in mind how you feel about the ruby implementation, what if we rework things a bit and just pass an `options` dictionary here? Otherwise, if you want to add another parameter here I can, however, I do feel like the parameter list length is already long as is... not a huge deal, just something to mind...

Reply:
> Just to check that the boolean flag is a boolean right?

Yep exactly.

> however, I do feel like the parameter list length is already long as is

Agreed

> what if we rework things a bit and just pass an options dictionary here?

That is one approach, and I think the quickest. But maybe it's also worth looking into using `**kwargs` for argument unpacking. I remember looking into that option for imgix-rb, but something about having already included `params={}` created usability issues. What do you think?

Reply:
Yeah, `**kwargs` has been on my mind, how much (if any) would you want to enforce the argument signature if we explore this route?